web/src/components/FilterColumnsModal.tsx

```diff
@@ -10,10 +11,31 @@ import {
   Checkbox,
   Button
 } from '@mui/material'
-import makeStyles from '@mui/styles/makeStyles'
 import { Clear } from '@mui/icons-material'
 import { ColumnLabel } from 'features/fbaCalculator/components/FBATable'
 
+const PREFIX = 'FilterColumnsModal'
```

Comment:
Just a general question on formatting a component as I'm pretty clueless... are there times when setting the style using the pattern  for the `ModalWindow` is preferred over something like this?

```
const StyledPaper = styled(Paper)(() => ({
  [`& .${classes.tableContainer}`]: {
    minWidth: '100%',
    overflowX: 'scroll',
    maxHeight: '72vh'
  }
}))
```

Reply:
I've noticed that these keyed class styles don't work well -- instead of investigating it most of the changes are single blocks of independent styles that can be composed.